### DISCUSSION TITLE
I want code for this side drawer

### DISCUSSION THREAD

**Discussion by anuragv218447** [ant-design/ant-design#31649] [Q&A]
![image](https://user-images.githubusercontent.com/88419375/128125511-55d8f194-0b7c-4ff4-b236-1eff59f1ad37.png)

**Comment by yoyo837** [comment]
Please send your resume to this email address directly.

**Reply by gaomigithub** [reply]
In my opinion, he was asking for the code which implemented this "side drawer" component XD

**Reply by yoyo837** [reply]
PR is welcome everywhere, anything, and anyone.

**Reply by anuragv218447** [reply]
> 
> 
> In my opinion, he was asking for the code which implemented this "side drawer" component XD

yes i want that only it was there in antd

**Reply by anuragv218447** [reply]
> 
> 
> PR is welcome everywhere, anything, and anyone.

yeah that i know but can u help me with this
